switchdev_utils/config-service/switchdev_network_config

```diff
@@ -0,0 +1,678 @@
+                 SRIOV_BIND_SERVICE_CONTENT)
+    run_command('systemctl', 'enable', 'sriov_bind')
+
+
+def enable_hw_offloaded(hw_offload_enabled=False):
+    logger.info("Enabling hardware offload for openvswitch..")
+    try:
+        run_command('/usr/bin/ovs-vsctl', 'set', 'Open_Vswitch', '.',
+                    'other_config:hw-offload=%s' % hw_offload_enabled)
+    except Exception as e:
+        logger.error("Failed to set hardware offload to %s\n%s" %
+                     (hw_offload_enabled, str(e)))
+        raise e
+    try:
+        run_command('systemctl', 'restart', 'openvswitch')
+    except Exception as e:
+        logger.error("Failed to restart openvswitch\n%s" % str(e))
+        raise e
+
+
+def write_ifcfg_files(ifcfg_interfaces_map):
+    logger.info("Writing ifcfg files for all net devices")
+    for interface, ifcfg_data in ifcfg_interfaces_map.items():
+        write_config(ifcfg_config_path(interface), ifcfg_data)
+
+
+def if_down_interface(device):
+    logger.info("Running /sbin/ifdown %s" % device)
+    try:
+        run_command('/sbin/ifdown', device)
+    except Exception as e:
+        logger.error("Failed to ifdown %s" % (device, str(e)))
+        raise e
+
+
+def if_up_interface(device):
+    logger.info("Running /sbin/ifup %s" % device)
+    try:
+        run_command('/sbin/ifup', device)
+    except Exception as e:
+        logger.error("Failed to ifup  %s" % (device, str(e)))
+        raise e
+
+
+def bring_interfaces_up(net_interfaces_list):
+    logger.info("Bringing interfaces down")
+    for interface in net_interfaces_list:
+        if_down_interface(interface)
+    logger.info("Bringing interfaces up")
+    for interface in net_interfaces_list:
+        if_up_interface(interface)
+
+
+def parse_version(version):
+    version_major = 0
+    version_minor = 0
+    try:
+        version_float = float(version)
+        version_major = int(version_float)
+        version_minor = int(str(version_float-int(version_float))[2:])
+    except Exception as err:
+        logger.error("configVersion is not provied in config.yaml file or is"
+                     " incorrect, Please provide configVersion as X.X\n%s" %
+                     str(err))
+        raise err
+    return version_major, version_minor
+
+
+def validate_config(config):
+    # validate_config is just a quick validation indicates if the user has
+    # typo or missing mandatory prams
+    # TODO//
+    # Define a schema for config.yaml file to do full config validation
+
+    sriov_pfs_list = []
+    bonds_list = []
+    version = config.get("configVersion")
+    version_major, version_minor = parse_version(version)
+    if version_major > VERSION_MAJOR:
+        err = "Provided Major version is upper than current one. Exiting.."
+        logger.error(err)
+        raise(RuntimeError(err))
+    if version_minor > VERSION_MINOR:
+        logger.debug("Provided Minor version is upper than current one, new"
```

Comment:
"Provided configuration version newer, new functionality may not be handled by this script"